### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Jacekey23**
CODE_OF_CONDUCT.md Standard
[aka.ms/opensource/moderation-support](https://aka.ms/opensource/moderation-support)

### CODE REVIEW

**Comment by Jacekey23** [comment]
can't tag in assignees.
@jiasli can you help to review and get it merged. Thanks

**Review by yonzhan — APPROVED** [review]

File: CODE_OF_CONDUCT.md
Comment (jiasli) [inline CODE_OF_CONDUCT.md:1]: Where is the source of this file?

Comment (Jacekey23) [reply CODE_OF_CONDUCT.md:1]: [the requirement](https://microsoft.sharepoint.com/teams/OpenSourceBlog/SitePages/Up.aspx?CID=4604f7ff-238e-2762-42d0-f9a138e4d312&xsdata=MDV8MDJ8fGUwYTJiMGZkMDM3MjQ0YjYwZjFlMDhkYzc4YmI4OTBifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2Mzg1MTc5OTQwMzMxODg2Mzh8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPak5oTlRaalkyVXhabVZoTmpRM09HVmhOekV3WkRjMFlqYzJOV1F3WVdSbFFIUm9jbVZoWkM1Mk1pOXRaWE56WVdkbGN5OHhOekUyTVRreE5ERTVORFV6fDc3YjZkY2YwZTcyMDQ2MmJhY2QxMDhkYzc4YmI4OTA4fGU0NzcwY2U0ZGE2MDQzZGY5MDAxZDFlZTRkZDMzNTFl&sdata=eVBpSEpnMUdwQkZQVHdiMzBIN0NKdk9qTGNxRTJjc2NtTW0xZ2o2am5EST0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cmingxuli%40microsoft.com&OR=Teams-HL&CT=1717579790843&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI0OS8yNDA1MDMwNzYxNiIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D#call-to-action)


the source of md: [link](https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md)

**Comment by jiasli** [comment]
Corresponding azure-cli repo PR: https://github.com/Azure/azure-cli/pull/29067

**Review by bebound — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Pull request merged**